client/time.h

```diff
@@ -25,10 +25,10 @@ extern "C" {
 extern xtime_t g_xdag_era;
 
 // returns a time period index, where a period is 64 seconds long
-xtime_t xdag_main_time(void);
+xdag_frame_t xdag_main_time(void);
```

Comment:
I think it makes sense to rename these function too.

Reply:
done